packages/component/src/SendBox/IconButton.js

```diff
@@ -2,21 +2,22 @@ import classNames from 'classnames';
 import PropTypes from 'prop-types';
 import React from 'react';
 
+import AccessibleButton from '../Utils/AccessibleButton';
 import useStyleSet from '../hooks/useStyleSet';
 
 const IconButton = ({ alt, children, className, disabled, onClick }) => {
   const [{ sendBoxButton: sendBoxButtonStyleSet }] = useStyleSet();
 
   return (
-    <button
-      className={classNames(sendBoxButtonStyleSet + '', className + '')}
+    <AccessibleButton
+      className={classNames(sendBoxButtonStyleSet + '', 'webchat__icon-button', className + '')}
       disabled={disabled}
-      onClick={onClick}
+      onClick={disabled ? undefined : onClick}
```

Comment:
Lol sure you don't want to memoize this? :)

Reply:
Yap, for two reasons:

1. It doesn't save any computation time, while using more memory
1. Given `disabled` and `onClick` are unchanged, re-render won't introduce unneeded changes (waste render)

I really think hard about it when you raise this question. On the other hand, I am thinking to memoize result from `classNames()`. 😂